ConceirgeDinning.Adapter.USRestaraunt/Models/Restaurant.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace ConceirgeDinning.Adapter.USRestaraunt.Models
+{
+    public class Restaurant
+    {
+        public Geo geo { get; set; }
+        public string hours { get; set; }
+        public Address address { get; set; }
+        public string restaurant_phone { get; set; }
+        public int restaurant_id { get; set; }
+        public string price_range { get; set; }
+        public List<object> menus { get; set; }
+        public int price_range_100 { get; set; }
```

Comment:
What is price_range_100?

Reply:
It is according to the contract provided by the supplier and it signifies about the prices restaurant charges